package.json

```diff
@@ -105,5 +105,8 @@
       "running": false
     }
   ],
-  "packageManager": "npm@10.9.2+sha512.8ab88f10f224a0c614cb717a7f7c30499014f77134120e9c1f0211ea3cf3397592cbe483feb38e0c4b3be1c54e347292c76a1b5edb94a3289d5448484ab8ac81"
+  "packageManager": "npm@10.9.2+sha512.8ab88f10f224a0c614cb717a7f7c30499014f77134120e9c1f0211ea3cf3397592cbe483feb38e0c4b3be1c54e347292c76a1b5edb94a3289d5448484ab8ac81",
+  "dependencies": {
+    "@vitejs/plugin-react": "^4.4.1"
+  }
```

Comment:
You don't need to add `@vitejs/plugin-react` as a dependency and should remove this.